classes/class-woothemes-sensei-certificates.php

```diff
@@ -1074,10 +1079,11 @@ public function certificate_link( $message, $course_id = 0 ) {
 		$view_link_profile  = Sensei()->settings->settings['certificates_view_profile'];
 		$is_viewable        = false;
 
-		if ( ( 'page' == get_post_type( $my_account_page_id )
+		if (  'page' == get_post_type( $my_account_page_id )
 				|| is_singular( 'course' )
-				|| isset( $wp_query->query_vars['course_results'] ) ) && $view_link_courses
-				|| isset( $wp_query->query_vars['learner_profile'] ) && $view_link_profile ) {
+				|| isset( $wp_query->query_vars['course_results'] )  && $view_link_courses
+				|| isset( $wp_query->query_vars['learner_profile'] ) && $view_link_profile
+		) {
```

Comment:
I've noticed some parenthesis missing and I'm out of clue what the old logic is doing, to be honest. 😆 Any ideas about what's going on?

If a beautify the old code, it looks like this:

```php
if (
	(
		'page' == get_post_type( $my_account_page_id )
		|| is_singular( 'course' )
		|| isset( $wp_query->query_vars['course_results'] )
	)
	&& $view_link_courses
	|| isset( $wp_query->query_vars['learner_profile'] )
	&& $view_link_profile
) {
```
Which doesn't make much sense to me. Maybe it's a bug? Do we need to make any changes to it at all?

Reply:
Yeah, this makes more sense. 👍

Reply:
Updated in 9fd5503

@m1r0 Please do give this a good look. Those parens are deep!

Reply:
Deep indeed! 😆 Ideally, we should break it up somehow but let's say it's good enough. 🙂